pom.xml

```diff
@@ -596,6 +596,11 @@
 				<artifactId>jaxb-impl</artifactId>
 			    <version>3.0.0-M4</version>
 			</dependency>
+			<dependency>
+				<groupId>org.springframework.data</groupId>
```

Comment:
Do you need this dependency

Reply:
Yes @sherrif10   Iam introducing  **spring-data-jpa** .
 Do you have a simpler way by which we can introduce **spring-data-jpa** ? or may be it's already available among the dependencies ?